code/game/objects/items/weapons/blades.dm

```diff
@@ -176,6 +176,31 @@
 	playsound(loc, 'sound/weapons/bladeslice.ogg', 25, 1)
 	return ..()
 
+
+/obj/item/weapon/knife/shiv
```

Comment:
This path is fucked up, `/obj/item/weapon/knife` doesn't exist. So you're defining it along with this. Suggest a repath to `/obj/item/tool/kitchen/knife/shiv`.